packages/slice-machine/tests/project/cypress/integration/0-is-running/00-onboarding.spec.js

```diff
@@ -37,12 +37,11 @@ describe('onboarding', () => {
     })
     cy.get('[data-cy=get-started]').click()
 
-    const continueSelector = '[data-cy=continue]'
-
-    const clickContinue = () => cy.get(continueSelector).click()
-    clickContinue()
-    clickContinue()
-    clickContinue()
+    const clickContinue = () => cy.get('[data-cy=continue]').click()
```

Comment:
maybe rename this tes or create another,
as when `is-onboarded` is in local-storage the user should see the first onboarding slide?

Reply:
No actually we don't want to have this logic, it's fine to see the first screen in any case